packages/modal/src/Modal.js

```diff
@@ -225,6 +227,7 @@ Modal.propTypes = {
   imgMode: PropTypes.bool,
   className: PropTypes.string,
   header: PropTypes.any,
+  timeout: PropTypes.number,
```

Comment:
I think this would be more appropriately named something like `animationDuration`; `timeout` makes me think the modal closes after this amount of time, or something.

Reply:
Timeout is a little confusing, it doesn't control the duration of the animation but the initial delay before the animation begins

Reply:
Ohhh gotcha, this makes sense then. Can you add a doc comment for that prop to avoid confusion?